run.sh

```diff
@@ -1,8 +1,11 @@
 poetry run python3 main.py \
     --model_name_or_path="openai/whisper-tiny" \
+    --dataset_config_path="conf/dataset_config.yaml" \
+    --train_dataset_fraction=0.3 \
+    --eval_dataset_fraction=0.1 \
```

Comment:
📝 Main change - config yaml for dataset is now arg settable as well as fractioned for testing the code on small dataset